contracts/Liquidator/Liquidator.sol

```diff
@@ -352,16 +352,16 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, Liqu
     function _reduceReservesInternal() internal {
         uint256 pendingReedemLength_ = pendingRedeem.length;
         if (pendingReedemLength_ == 0) return;
-        for (uint256 index = pendingReedemLength_ - 1; index >= 0; index--) {
-            address vToken = pendingRedeem[index];
+        for (int256 index = int256(pendingReedemLength_) - 1; index >= 0; index--) {
```

Comment:
Let's say `pendingReedemLength_` is high enough so we consume 100% of the gas in the loop. Would it make sense to cap this value in this for-loop, so we would process always `min(10, pendingReedemLength_)`, for example. This way we would ensure we reduce some reserves in each interaction